cmd/zb/perf.go

```diff
@@ -307,13 +307,38 @@ type testFunc func(
 	client *resty.Client,
 ) error
 
+// nolint:gosec
 func GetCatalog(
 	workdir, url, repo string,
 	requests int,
 	config testConfig,
 	statsCh chan statsRecord,
 	client *resty.Client,
 ) error {
+	var repos []string
+
+	var err error
+
+	sizes := [3]int{
+		smallBlob,
+		mediumBlob,
+		largeBlob,
+	}
+
+	mrand.Seed(time.Now().UTC().UnixNano())
+	imagesNo := mrand.Intn(requests) + 1
+
+	for imageNo := 0; imageNo < imagesNo; imageNo++ {
```

Comment:
This is unpredictable, so how do I now compare against other registries or across runs.
Would say, make it 70-20-10 (small-medium-large) split.